### DISCUSSION TITLE
Detecting connector in a Fabric mod

### DISCUSSION THREAD

**Discussion by zenfyrdev** [Sinytra/Connector#41] [Q&A] [answered]
Hello again 😅

As the title says, I want to detect connector to force enable/disable certain features.

Can I rely on 

```java
FabricLoader.getInstance().isModLoaded("connectormod");
```
Or there's something else I also need to check?

**Comment by Su5eD** [comment, marked as answer]
That check should be absolutely fine. Any sort of entrypoint a mod might have (preLaunch, main, mixin config) is only invoked after we have loaded all mod information into fabric loader.

**Reply by zenfyrdev** [reply]
Alright, thank you!
